planning/planning_scenarios_distributed_db.adoc

```diff
@@ -0,0 +1,59 @@
+=== Container Interconnection: Distributed Database Server
+
+Introduce the basic environment and began to foreshadow the perceieved complications
+
+==== Database server
+
+By their very nature, distributed application components need to communicate with one another.  
+Container technologies encourage developers to make these interconnection points explicit and 
+provide a number of mechanisms to coordinate or otherwise enable communication between containers.
+
+Consider the database example in the previous section.  Once we have established persistent 
+storage for the database server, we also need to consider how database clients will connect to it.
+In nearly all cases these connections will occur through a socket, ever over the network or locally
+via UNIX domain socket special file.
+
+(Diagram placeholder - Block for running container, inset for listening port on top of container 
+block, distinct block outside of container showing shared/mapped directories for UNIX sockets.)
+
+Simple non-distributed applications may assume that a database is co-located on the same server 
+and use an established port number, or UNIX domain socket path, as their default access mechanism.
+
+True multi-node distributed applications may host the database as a distinct node in which case 
+communication must occur via the network.  Clients that wish to use the database must be made 
+aware of its location, either via explicit configuration or a service discovery mechanism.
+
+(diagram placeholder - single node versus multi node with IP address)
+
+==== Container environment - Docker
+
+Once an interconnection mechanism has been established a container developer must ensure 
+that service containers are properly configured to allow access to these connections.
+
+For network connections, Docker provides a simple directive in the Dockerfile to expose a port from the 
+running container.  For example, to create a Postgres DB server container that listents on the default 
+Postgres port, you would add the following line:
+
+EXPOSE 5432
```

Comment:
We did indeed, apologies  - Those would be, I think, Network details in "4.8. Network Considerations" (which is currently empty in my branch) and Storage details in "4.9. Storage Considerations" (which is not)